src/logger.h

```diff
@@ -1,4 +1,38 @@
 #pragma once
+
+#include <fstream>
+
+#include <boost/log/core/core.hpp>
+#include <boost/log/sources/global_logger_storage.hpp>
+#include <boost/log/sources/logger.hpp>
 #include <boost/log/trivial.hpp>
+#include <boost/log/utility/setup/from_stream.hpp>
+
+namespace sdllog {
+namespace logging = boost::log;
+namespace src = boost::log::sources;
+using LoggerType = src::logger_mt;
+
+#define SEVERITY_THRESHOLD logging::trivial::warning
+
+BOOST_LOG_GLOBAL_LOGGER(logger,
+                        boost::log::sources::severity_logger_mt<
+                            boost::log::trivial::severity_level>)
+
+#define LOG(severity) \
+  BOOST_LOG_SEV(logger::get(), boost::log::trivial::severity)
```

Comment:
@AGaliuzov, how this works? It gets some default logger? I think we should pass the logger. What do you think?